### PR TITLE
Bump dependencies for all dotnet examples for v2

### PR DESCRIPTION
**Pull request by jaxxstorm**
This PR only deals with dotnet dependencies.

Draft because we need to wait for these to be published

### CODE REVIEW

**Comment by mikhailshilkov** [comment]
@jaxxstorm I went ahead with https://github.com/pulumi/examples/pull/648 and updated to beta's because there are non-trivial code changes required to go with that. Update to release should be just a version bump.

File: aws-cs-eks/Aws.EksCluster.csproj [hunk 1]
```diff
@@ -7,8 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Pulumi.Aws" Version="1.30.0-preview" />
-    <PackageReference Include="Pulumi.Kubernetes" Version="1.6.0-preview" />
+    <PackageReference Include="Pulumi.Aws" Version="2.0.0" />
+    <PackageReference Include="Pulumi.Kubernetes" Version="2.0.0" />
```
Comment (mikhailshilkov) [inline aws-cs-eks/Aws.EksCluster.csproj:11]: We should probably use wildcard versions, `2.*` etc which would resolve to the highest `2.x.x` stable release

File: digitalocean-cs-k8s/Digitalocean.K8s.csproj [hunk 2]
```diff
@@ -9,8 +9,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Pulumi.DigitalOcean" Version="1.7.0-preview" />
-    <PackageReference Include="Pulumi.Kubernetes" Version="1.5.8-preview" />
+    <PackageReference Include="Pulumi.DigitalOcean" Version="2.0.0" />
+    <PackageReference Include="Pulumi.Kubernetes" Version="2.0.0" />
```
Comment (mikhailshilkov) [inline digitalocean-cs-k8s/Digitalocean.K8s.csproj:13]: Missed this one?

File: digitalocean-cs-loadbalanced-droplets/Digitalocean.LoadbalancedDroplets.csproj [hunk 3]
```diff
@@ -9,7 +9,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Pulumi.DigitalOcean" Version="2.0.0-beta.1" />
+    <PackageReference Include="Pulumi.DigitalOcean" Version="2.*" />
+    <PackageReference Include="Pulumi.Kubernetes" Version="2.*" />
```
Comment (mikhailshilkov) [inline digitalocean-cs-loadbalanced-droplets/Digitalocean.LoadbalancedDroplets.csproj:13]: I actually think K8s isn't needed here

File: digitalocean-cs-k8s/Digitalocean.K8s.csproj (hunk 2 above)
Comment (jaxxstorm) [reply digitalocean-cs-k8s/Digitalocean.K8s.csproj]: Yep, fixed!

File: digitalocean-cs-loadbalanced-droplets/Digitalocean.LoadbalancedDroplets.csproj (hunk 3 above)
Comment (jaxxstorm) [reply digitalocean-cs-loadbalanced-droplets/Digitalocean.LoadbalancedDroplets.csproj]: Removed

**Pull request merged**